documentation/asciidoc/topics/con_hotrod_clients.adoc

```diff
@@ -6,16 +6,25 @@ Hot Rod is a binary TCP protocol that {brandname} provides for high-performance
 
 .Client intelligence
 
-The Hot Rod protocol includes a mechanism that provides clients with an up to date view of the cache topology.
+The Hot Rod protocol includes a mechanism that provides clients with an up-to-date view of the cache topology.
 Client intelligence improves performance by reducing the number of network hops for read and write operations.
 
 Clients running in the same {k8s} cluster can access internal IP addresses for {brandname} pods so you can use any client intelligence.
-The default intelligence, `HASH_DISTRIBUTION_AWARE`, is recommended because it allows clients to route requests to primary owners, which improves performance.
 
 [NOTE]
 ====
 {k8s} cluster administrators can define network policies that restrict traffic to {brandname}.
 In some cases network isolation policies can require you to use `BASIC` intelligence even when clients are running in the same {k8s} cluster but a different namespace.
 ====
 
-Hot Rod clients must use `BASIC` intelligence when connecting to {brandname} through a `LoadBalancer`, `NodePort`, or {openshiftshort} `Route`.
+`HASH_DISTRIBUTION_AWARE`:: The default intelligence mechanism. Best mechanism for allowing clients to route requests to primary owners, which improves a client's performance. 
+
+[NOTE]
+====
+Hot Rod clients must use `BASIC` intelligence in the following situations:
+
+* You connect to {brandname} through a `LoadBalancer`, `NodePort`, {openshiftshort} `Route`, or {k8s} Ingress.
+* A client running external to {k8s} cluster cannot access IP addresses for {brandname} pods in this cluster. 
```

Comment:
Tbh I don't think we need to explicitly state this second scenario, as an external client must always connect via one of the three methods detailed in the first bullet point as k8s services are not accessible externally.

Reading the Jira it seems the issue is specifically focused on the xsite scenario where the client is deployed on one k8s cluster but fail-over to a different site (on a seperate k8s cluster) is possible. In which case, can we just have something as simple as:

"Cross-site failover is required"

Reply:
Thanks, @ryanemerson . I revised the sentence. Does the use case read better now?